src/components/Planet/PlanetStats/PlanetStats.jsx

```diff
@@ -0,0 +1,21 @@
+import PlanetStatItems from './PlanetStatsItem/PlanetStatItem';
+import './PlanetStats.css';
+
+const PlanetStats = ({ stats }) => {
+  return (
+    <div className="PlanetStat">
+      <PlanetStatItems
+        statType="rotation time"
+        statValue={stats.rotationTime}
+      />
+      <PlanetStatItems
+        statType="revolution time"
+        statValue={stats.revolutionTime}
+      />
+      <PlanetStatItems statType="radius" statValue={stats.radius} />
```

Comment:
Use the same convention every time you call the component. After that, merge the PR